### PR TITLE
[Reviewer - Andy] Include deb instruction in readme

### PR DESCRIPTION
**Pull request by johadalin**
Does this seem reasonable and correct?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,3 +48,9 @@ group:          files
 shadow:         files ato
 ```
 
+For debian packages, run the following
```
Comment (bossmc) [inline README.md]: Debian is capitalized (it's short for Debbie and Ian :grinning:).

File: README.md [hunk 2]
```diff
@@ -48,3 +48,9 @@ group:          files
...
+```console
+fakeroot debian/rules binary
+```
+
+This should create the correct .deb in the home directory
```
Comment (bossmc) [inline README.md]: It makes it in the parent directory (not necessarily the home directory).

Comment (bossmc) [inline README.md]: Also I'd put `.deb` in backticks for formatting.

**Comment by bossmc** [comment]
I think this would be better as "installing from source" and "Installation from debian packages" subsections.

**Comment by johadalin** [comment]
looking better?

**Pull request merged**